flow/project.py

```diff
@@ -1922,6 +2226,19 @@ def _add_job_selection_args(cls, parser):
             nargs='+',
             help="Only select jobs that match the given document filter.")
 
+    @classmethod
+    def _add_group_selection_args(cls, parser):
```

Comment:
I'd rather have groups work exactly like other operations, but they are internally expanded essentially. Could you provide some insight on why you think that we need a separate top-level option for this?